arm-storsimple8000series/2017-06-01/swagger/storsimple.json

```diff
@@ -1575,12 +1773,36 @@
         ],
         "parameters": [
           {
+            "name": "deviceName",
```

Comment:
Similar comment as I made in another instance, you may want to move these parameters down, as code generation will match this order, and in general it's good to have the method signature follow path parameters order. 
Please check other operations for this as well.